### PR TITLE
Added patch for py35

### PR DESCRIPTION
**Pull request by pmlandwehr**
Doesn't work locally, but can't think why. Want to test against CI in case my local caches are breaking things...

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,11 +11,11 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   {{ hash_type }}: {{ hash_val }}
-  # patches:
-  #   - python35-compat.patch  # [py35]
+  patches:
+     - python35-compat.patch  # [py35]
```
Comment (pelson) [inline recipe/meta.yaml]: Have you made this patch available upstream? I'm sure they would love to know about it! Providing the link as a comment will also allow us to cross-reference when we upgrade the recipe to determine if the patch is no longer needed.

Comment (pmlandwehr) [inline recipe/meta.yaml]: I actually stole this patch from upstream - it's mentioned on some issue mailing lists; see [here](https://bitbucket.org/catherinedevlin/cmd2/issues/18/python-35-renames-subprocessmswindows)

Comment (pelson) [inline recipe/meta.yaml]: Ah, OK, thanks @pmlandwehr. If you wouldn't mind, I think it would be healthy to comment the this line with this information.

**Comment by pmlandwehr** [comment]
I can make that change once the new py35 version propagates - I'm building a dependent package and don't want to wrangle with the rebuilding yet.

**Pull request merged**